app/services/rapid_ready_for_decision/processor_selector.rb

```diff
@@ -6,38 +6,40 @@ def initialize(form526_submission)
       @form526_submission = form526_submission
     end
 
+    RrdConstants = RapidReadyForDecision::Constants
+
+    DISABILITY_TO_PROCESSOR_CLASS = {
+      hypertension: RapidReadyForDecision::DisabilityCompensationJob,
+      asthma: RapidReadyForDecision::Form526AsthmaJob
+    }.freeze
+
     def processor_class
       return unless rrd_enabled?
 
       # single-issue claims only
-      return unless disability_types.size == 1
+      return unless mapped_submission_disabilities.size == 1
 
-      disability_type = disability_types.first
+      disability_symbol = mapped_submission_disabilities.first
       # must be a disability supported by RRD
-      return unless disability_type
+      return unless disability_symbol
 
+      disability_struct = RrdConstants::DISABILITIES[disability_symbol]
```

Comment:
Given that `disability_symbol` is only used here to get the struct back, and below to get the class (which could also live in the struct) I don't really see the benefit of switching from struct to symbol. Maybe I missed something -- are we more worried about passing structs around, rather than symbols?

Reply:
I am somewhat worried about passing around a struct -- it seems like asking for trouble. Comparing symbols is straight-forward and fast; comparing structs is not.

Reply:
I'm also planning to pass the disability symbols around to various classes (that are currently hypertension-specific) so that we can make them more general.